topics/imaging/tutorials/hela-screen-analysis/tutorial.md

```diff
@@ -95,7 +101,7 @@ The dataset required for this tutorial contains a screen of DAPI stained HeLa nu
 > 9. Rename {% icon galaxy-pencil %} the collection to `PLK1`
 > 9. Upload {% icon galaxy-upload %} the following segmentation filter rules as a new pasted file (format: `tabular`):
 >    ```
->    area	eccentricity
+>    	area	eccentricity
```

Comment:
that looks wrong here. the first one

Reply:
Why? The file has three columns. The first column has no heading.

Reply:
You are correct. I have seen two ... but it seems to be just one.

:+1: from me.